### ISSUE TITLE
[Feature Request]: Adding a "Citing and Logo Assets" Page

### ISSUE THREAD

**Issue by ranggakd** [pola-rs/polars#9369]
### Problem description

Dear Polars developers and community,

I am a user of Polars and appreciate the great work you've put into this package. To help spread the word about Polars, I would like to properly cite it in my work and use the Polars logo when promoting its use. However, I was unable to find official citation information or logo assets on the Polars website. 

In comparison, the Pandas project offers a dedicated page (https://pandas.pydata.org/about/citing.html) which provides citation information and logo assets for users and advocates. I believe adding a similar page for Polars would be beneficial for the community for the following reasons:

1. **Citation Info**: Providing a standard way to cite Polars can help track its use in academia and industry, increasing its visibility and demonstrating its impact.

2. **Logo Assets**: Offering official, high-quality logo assets can help users correctly represent Polars in presentations, blog posts, papers, and other materials.

I understand the Polars project is licensed under the MIT license, which permits the use and distribution of the software. It is unclear to me if this license also covers logo usage or if a separate policy is needed.

If the community agrees this feature would be beneficial, I'm happy to contribute towards its realization in any way I can. I believe a dedicated "Citing and Logo Assets" page would be a great resource for all users and advocates of Polars.

If you agree, please upvote this issue to get the attention it deserves. Your vote could help expedite the process and make Polars even better!

Thank you for considering my proposal.

Best regards,
Rangga

**Comment by sm-Fifteen** [comment]
There's [a bit of that](https://github.com/pola-rs/polars-static/tree/master/logos) in the repository containing the logos.

> ### Please keep the use of the Polars name and logo fair!
> 
> Projects and documents that want to include and/or contribute to Polars should link to the GitHub [organization](https://github.com/pola-rs) and/or [website](https://pola.rs/), and use the official [logo](https://github.com/pola-rs/polars-static/tree/master/logos).

**Comment by mcrumiller** [comment]
See also #7226

**Comment by ranggakd** [comment]
> There's [a bit of that](https://github.com/pola-rs/polars-static/tree/master/logos) in the repository containing the logos.

I appreciate the existing efforts to include logo information within the repository. @sm-Fifteen 

> See also https://github.com/pola-rs/polars/issues/7226

Also, I appreciate the existing efforts to include citations via GitHub-Zenodo integration. @mcrumiller 

However, I strongly emphasize the need for a dedicated citation and logo assets page on the Polars website.

A centralized, easy-to-access page would immensely streamline the user experience, as it prevents the need for extensive navigation or reliance on SEO. The solution proposed in [#7370](hhttps://github.com/pola-rs/polars/pull/7370) partially addresses the citation issue, yet it remains within GitHub, not the website. I urge consideration of the increased visibility, accessibility, and user-friendliness this dedicated page would provide to Polars' community.

**Comment by stinodego** [comment]
Good idea. We should pick this up when overhauling our website. There are a lot of things we can do to improve communication towards the community, this is one of them.

In the meantime, please refer to the resources linked above.

**Comment by Romain-Thomas-Shef** [comment]
If I can add a comment, adding a CITATION.cff and/or CITATION.bib in the repository will allow users to use the 'cite this repository' functionality directly from the repo page: 

<img width="209" height="119" alt="Image" src="https://github.com/user-attachments/assets/9faf252d-179c-4dd5-9770-a7b385509787" />
